test/chat_8_3_0_0_state_machine.py

```diff
@@ -77,6 +75,8 @@ def create_room(self, creator, room_name):
             network_create_room_event = self.network[creator].chat[CHAT_VERSION].create_room(room_name=room_name)
             room_channel = network_create_room_event['room']['channel']
             model_create_room_event = self.model.create_room(creator, room_channel, room_name)
+            print(network_create_room_event)
```

Comment:
I don't think we would want to keep the print of events in a production app.

Reply:
Oops! Apologies